README.md

```diff
@@ -51,18 +120,19 @@ Without a prefix specified, the default `fas` is assumed:
 {{fa-icon 'square'}}
 ```
 
-If you need to distinguish styles between two different icons of the same name that have both
-been added to the library, use `prefix=`. 
+If you need to distinguish styles between two different icons of the same name
+that have both been added to the library, use `prefix=`.
```

Comment:
This text isn't really correct. Since the library defaults to `fas`, you can't use icons from any other style unless you specify the prefix. More correct text might be:

> If you want to use an icon from a different style, use `prefix=`

Reply:
Gotcha. Will fix.